### DISCUSSION TITLE
为什么会有这样的黑框框。

### DISCUSSION THREAD

**Discussion by Dysnyma** [flxzt/rnote#1329] [Development]
<img width="461" alt="image_38" src="https://github.com/user-attachments/assets/dff4fc1d-688c-4ced-ac1c-a39a5ea87d85" />

**Comment by Doublonmousse** [comment]
That's issue #842.
This is an issue of the gui toolkit we are using (gtk4) and there's a tracking issue there.
It may get fixed with the next release of the toolkit (gtk 4.18)

**Reply by Dysnyma** [reply]
![image](https://github.com/user-attachments/assets/4bd36178-957d-4dcd-96fb-3da002f9bea1)
非常感谢您的回复，我发现使用集成显卡就不会出现这样的问题。

**Reply by Dysnyma** [reply]
![image](https://github.com/user-attachments/assets/b8ab145b-c61c-4750-8056-1d1fd7db0789)
我发现第一次打开会有黑框框，第二次一般不会。
